plugins/grpc-ecosystem/gateway/v2.21.0/buf.plugin.yaml

```diff
@@ -0,0 +1,23 @@
+version: v1
+name: buf.build/grpc-ecosystem/gateway
+plugin_version: v2.21.0
+source_url: https://github.com/grpc-ecosystem/grpc-gateway
+integration_guide_url: https://github.com/grpc-ecosystem/grpc-gateway#usage
+description: gRPC to JSON proxy generator following the gRPC HTTP spec.
+output_languages:
+  - go
+registry:
+  go:
+    min_version: "1.20"
+    deps:
+      - module: github.com/grpc-ecosystem/grpc-gateway/v2
+        version: v2.21.0
+  opts:
+    - paths=source_relative
+    - standalone=true
+    - separate_package=true
+deps:
+  - plugin: buf.build/protocolbuffers/go:v1.34.1
```

Comment:
```suggestion
  - plugin: buf.build/protocolbuffers/go:v1.34.2
```

Reply:
This should be okay because the minimum version was bumped to 1.20 in 

https://github.com/grpc-ecosystem/grpc-gateway/commit/be89844fba28c0f14bdb84a71d9c03a15d4e6e6f